spec/requests/registration_spec.rb

```diff
@@ -28,7 +28,7 @@
     let(:doc) { instance_double(Prawn::Document, render: '') }
 
     it 'generates a tracking sheet with the right default name' do
-      get "/registration/tracksheet?druid=#{bare_druid}", headers: headers
+      get "/registration/tracksheet?druid=#{druid}", headers: headers
```

Comment:
I don't see how this is proving that it's a druid without a namespace - should it prove that?